src/Microsoft.DotNet.Wpf/specs/tooltip.md

```diff
@@ -0,0 +1,305 @@
+- **Implementation**.  Internal-facing spec - details needed by implementers, testers, and reviewers, but not part of the public documentation.
+- **Discussion**.  Remarks that aren't part of the spec, but help understand it.  Including how the spec meets the requirements, goals and non-goals, alternatives that were considered, etc.  
+- **Summary of behavior changes**.  Consolidated in one place for convenience.
+- **API Details**.  Consolidated in one place for convenience.
+
+
+# Requirements 
+This spec cites the following sources (the [brackets] contain abbreviations used elsewhere in this document):
+
+1. WCAG 2.1, section 1.4.13: [Content on hover or focus](https://www.w3.org/TR/WCAG21/#content-on-hover-or-focus) [WCAG] 
+2. WCAG 2.1:  [Understanding 1.4.13](https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html)   [WCAG]
+3. MAS 1.4.13:  [Content on hover or focus](https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7B56729308-859e-4b32-ab27-04d4364b3003%7D&action=view&wdLOR=c9C9640FB-F04E-401A-BF46-A79887F322D3&wdAccPdf=0&wdparaid=38854747) [MAS]
+4. The [Figma diagram](https://www.figma.com/file/8LeT0Eq7a3ylAyM19c0FJp/ASC%3A-Tooltip?node-id=110%3A0) (as of February 2021) [FD]
+5. Email thread (MS internal) [ET] 
+6. WPF [Tooltip Overview](https://docs.microsoft.com/en-us/dotnet/desktop/wpf/controls/tooltip-overview?view=netframeworkdesktop-4.8) (and related links) [TO]
+
+The first three describe behavior in general terms, guiding any UI platform (WCAG) or Microsoft UI platforms (MAS).
+The Figma diagram has more specific requirements, but is not a complete spec.
+The email thread among many relevant MS teams helped fill in omissions, corrected errors, and clarified ambiguities in the diagram.
+The Tooltip Overview documents the existing behavior in WPF as of .NET 4.7.2.
+
+# Applicability
+
+All the changes described here are required to appear in .NET 6.0.
+
+The behavior changes will also appear in servicing updates for earlier versions as follows:
+- .NET 4.6-4.7.2, .NET 4.8:  the next biannual accessibility update (expected summer 2021)
+- .NET Core 3.1, .NET 5.0:  same time frame as above, patch version TBD
+
+The API changes (and behavior that depends on them) are only in .NET 6.0;
+servicing updates cannot include changes to public API.
+
+In the servicing updates, the new behavior is opt-in by setting
+`Switch.UseLegacyAccessibilityFeatures.5` to false, as described [here](https://docs.microsoft.com/en-us/dotnet/framework/whats-new/whats-new-in-accessibility).
+
+
+
+# Conceptual pages (How To)
+
+## Declaring a tooltip
+You can declare a tooltip in XAML or code as described in [TO].
+Starting in .NET 6.0, your declarations can also use the properties
+`ToolTip.ShowsToolTipOnKeyboardFocus`, and
+`ToolTipService.ShowsToolTipOnKeyboardFocus`.
+
+## Opening a tooltip
+When a parent element declares a tooltip, there are four ways to open (show) the tooltip:
```

Comment:
What does parent mean? Does this mean the element to which the attached ToolTip property was attached?

Reply:
Yes.  This is the term used in several of the cited docs.

Well, your question is a little garbled.  Parent means "the element that owns the tooltip".  In WPF terms, it's the element whose `ToolTip` property is non-null (that property is available on `FrameworkElement` and `FrameworkContentElement`, it's a normal property, not an attached property).  I assume that's what you were trying to say.